src/components/Calendar/Calendar.cy.tsx

```diff
@@ -72,9 +80,68 @@ describe("<Calendar />", () => {
         );
     });
 
-    it("shows description when event is clicked", () => {
+    it("shows description when event with description is clicked", () => {
+        cy.mount(<Calendar initialEvents={sampleEvents} />);
+        cy.get(".fc-event-title").contains("event2").click();
+        cy.get(".MuiDialog-container").should("include.text", "description");
```

Comment:
is this testing that the word heading `Description: ` or the text `a description` is shown? Potentially change the test text to be not exactly the same as the heading :)

Reply:
(make sure you get all the new tests for this!)

Reply:
refactored as suggested